.github/CODEOWNERS

```diff
@@ -1 +1 @@
-* @benedeki @miroslavpojer @lsulak @salamonpavel
```

Comment:
This is an OSS repo, I'm thinking about whether we should keep David here on this list since he even recently reacted on this PR. He still might have interest to contribute to this project